MergerCli/Program.cs

```diff
@@ -93,8 +95,7 @@ private static void Main(string[] args)
                 bool validate = config.GetConfiguration<bool>("GENERAL", "validate");
                 for (int i = 1; i < sources.Count; i++)
                 {
-                    Stopwatch stopWatch = new Stopwatch();
-                    stopWatch.Start();
+                    Stopwatch stopWatch = Stopwatch.StartNew();
```

Comment:
Once you added another `Stopwatch` variable to this function there is a need for a clearer name for this one.

Reply:
changed to mergeRunTimeStopwatch